site/book/05-developing-functions/02-developing-in-Go.md

```diff
@@ -20,6 +20,7 @@ following features:
 ```shell
 $ go mod init github.com/user/repo
 $ go get sigs.k8s.io/kustomize/kyaml@v0.10.6
+$ go get ./...
```

Comment:
is this actually needed? It worked for me without it.

Reply:
I had it complain but my guess is if you do it once maybe you got packages on your local machine.  I am flex.  We can leave it out.  All the people who know GoLang post 101 will know what to fill in.

Reply:
Close the PR if you don't think it's necessary.